components/automate-ui/src/app/pages/project/rules/project-rules.component.ts

```diff
@@ -72,48 +75,47 @@ export class ProjectRulesComponent implements OnInit, OnDestroy {
 
       combineLatest(
         this.store.select(getStatus),
-        this.store.select(updateStatus)
+        this.store.select(updateStatus),
+        this.store.select(getProjectStatus)
       ).pipe(
         takeUntil(this.isDestroyed),
-        map(([gStatus, uStatus]) => {
+        map(([gStatus, uStatus, gpStatus]: [string, string, string]) => {
```

Comment:
For future reference, just realized one could also just use `string[]`, too. 😁

Reply:
💭 `string[]` is wider than `[string, string, string]`, though -- `["a", "b", "c", "d"]` is in the former, not the latter. 🤷‍♂